website/src/content/development/layout-guidelines.md

```diff
@@ -0,0 +1,122 @@
+# Layout Guidelines
+
+Layouts are the foundation for interactive experiences. They affect the way data is structured and displayed. The different layout types set the minimum and maximum width of a container and its position on the page.
+
+### Design
+
+In commercetools, we always try to make better use of the space when building Merchant Center applications. As a result, we provide guidelines and rules that are recommended to be followed when developing applications, both by internal and external engineering teams. You can choose from 5 different layout options to display content in the Merchant Center:
+
+#### Narrow
+
+* Single Column narrow should be used for simple forms.
+* The form can be split into multiple sections by collapsible panels, cards, or headlines.
+
+IMAGE
+
+#### Wide (Single Column)
+* Single column wide should be used for content where more space is necessary.
```

Comment:
I would suggest to mention this content layout will horizontally center the content.